macros/edr/data_monitoring/anomaly_detection/get_anomaly_scores_query.sql

```diff
@@ -193,4 +197,8 @@
 
     {% endset %}
     {{ return(anomaly_scores_query) }}
-{% endmacro %}
\ No newline at end of file
+{% endmacro %}
+
+{% macro get_negative_value_supported_metrics() %}
+    {% do return(["min", "max", "average", "standard_deviation", "variance", "sum"]) %}
```

Comment:
@elongl just to better understand- this will prevent negative row count from showing because count is not in this list?